docker.version

```diff
@@ -5,8 +5,8 @@
 CONTAINER_BUILDVERSION_MAJOR=1
 CONTAINER_BUILDVERSION_MINOR=0
 CONTAINER_BUILDVERSION_PATCH=0
-CONTAINER_BUILDVERSION_BUILDNR=39
-CONTAINER_BUILDVERSION_DATE=20210915
+CONTAINER_BUILDVERSION_BUILDNR=41
```

Comment:
should this be 40?

Reply:
we already released 40 and are on 41 now